File: Desarrollos AddOn SAP B1/Samples/COM DI/CSharp/04.SerialAndBatch/globalD.cs
using System.Diagnostics;
using Microsoft.VisualBasic;
using System;
using System.Data;
using System.Drawing;
using System.Windows.Forms;
using System.Collections;

////  SAP MANAGE DI API 2007 SDK Sample
////****************************************************************************
////
////  File:      globalD.vb
////
////  Copyright (c) SAP MANAGE
////
//// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
//// ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
//// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
//// PARTICULAR PURPOSE.
////
////****************************************************************************
namespace WindowsApplication2
{
	sealed class globalD
	{
		
		public static SAPbobsCOM.Company oCompany = new SAPbobsCOM.Company();
		public static SAPbobsCOM.Items oItems;
		public static SAPbobsCOM.Documents oInvGenEntry;
		public static SAPbobsCOM.Documents oInvGenExit;
		public static string ErrMsg;
		public static int ErrCode;
		public static int RetVal;
		public static SAPbobsCOM.BoYesNoEnum RetBool;
		public static bool flagManage;
		public static bool flagRelease;
		public static bool flagForce;
		public static bool flagBatch;
		public static bool flag;
		public static string tempStr = null;
		public static string manSerialNumber;
		public static string manBatchNumber;
		public static int totalNumber;
		public static string batchNum;
		public static string interSerNumber;
	}
	
	
}

File: Desarrollos AddOn SAP B1/Samples/COM DI/VB.NET/04.SerialAndBatch/globalD.vb
'//  SAP MANAGE DI API 2007 SDK Sample
'//****************************************************************************
'//
'//  File:      globalD.vb
'//
'//  Copyright (c) SAP MANAGE
'//
'// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF
'// ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO
'// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A
'// PARTICULAR PURPOSE.
'//
'//****************************************************************************
Option Strict Off
Option Explicit On 
Module globalD

    Public oCompany As New SAPbobsCOM.Company
    Public oItems As SAPbobsCOM.Items
    Public oInvGenEntry As SAPbobsCOM.Documents
    Public oInvGenExit As SAPbobsCOM.Documents
    Public ErrMsg As String
    Public ErrCode As Integer
    Public RetVal As Integer
    Public RetBool As SAPbobsCOM.BoYesNoEnum
    Public flagManage As Boolean
    Public flagRelease As Boolean
    Public flagForce As Boolean
    Public flagBatch As Boolean
    Public flag As Boolean
    Public tempStr As String
    Public manSerialNumber As String
    Public manBatchNumber As String
    Public totalNumber As Integer
    Public batchNum As String
    Public interSerNumber As String
End Module

